library/src/main/java/com/datatorrent/lib/db/cache/CacheManager.java

```diff
@@ -210,6 +272,54 @@ public String getRefreshTime()
     Map<Object, Object> loadInitialData();
   }
 
+
+  /**
+   * Used by {@link CacheManager} to pass properties to its stores which implement {@link Component}.
+   */
+  public static class CacheContext implements Context
+  {
+    public static final transient Attribute<Boolean> READ_ONLY_ATTR = new Attribute<>((false));
+    public static final transient Attribute<Integer> NUM_INIT_CACHED_LINES_ATTR = new Attribute<>((-1));
```

Comment:
I see, it would slightly change the behavior, e.g. in FSLoader it has no upper bound if num_init_cached_lines_attr was not set. With the change Integer.MAX_VALUE would always be the default limit there